pytest_html/extras.py

```diff
@@ -10,10 +10,10 @@
 FORMAT_VIDEO = "video"
 
 
-def extra(content, format, name=None, mime_type=None, extension=None):
```

Comment:
I think the rebase is a little off. This change already exists in master ( see https://github.com/pytest-dev/pytest-html/blob/master/pytest_html/extras.py#L13 ). It looks like this PR counts the merge of master as your changes